.github/labels.yml

```diff
@@ -39,6 +39,7 @@ infra :building_construction::
   - ".github/**"
   - "LICENSE"
   - "package*"
+  - "!test/**"
```

Comment:
This is unnecessary, as `/*.js` only matches JavaScript files in the root of the repository.